test/Spago/Build.purs

```diff
@@ -32,6 +32,12 @@ spec = Spec.around withTempDir do
       spago [ "init", "--name", "aaa", "--use-solver" ] >>= shouldBeSuccess
       spago [ "build" ] >>= shouldBeSuccess
 
+    Spec.after (rmRf <<< _.testCwd)
+      $ Spec.it "exits when purs exits non-ok"
+      $ \{ spago, fixture } -> do
+          spago [ "init", "--name", "aaa" ] >>= shouldBeSuccess
+          spago [ "build", "--purs-args", "--non-existent" ] >>= shouldBeFailureErr (fixture "purs-not-ok.txt")
```

Comment:
A couple of things:
- we can remove the `rmRf` for the temp dir since we don't need it anymore
- we can patch the fixture using the `sanitize` hook in the fixture machinery, so that we can get the Windows CI happy as well 🙂 

```suggestion
    Spec.it "exits when purs exits non-ok" \{ spago, fixture } -> do
      spago [ "init", "--name", "aaa" ] >>= shouldBeSuccess
      spago [ "build", "--purs-args", "--non-existent" ] >>=
        checkOutputs'
          { stdoutFile: Nothing
          , stderrFile: Just (fixture "purs-not-ok.txt")
          , result: isLeft
          , sanitize: String.trim >>> String.replace (String.Pattern "Usage: purs.bin") (String.Replacement "Usage: purs")
          }
```